meta-tedge/recipes-tedge/tedge/tedge-services.inc

```diff
@@ -3,13 +3,17 @@ SRC_URI += "git://git@github.com/thin-edge/tedge-services.git;protocol=https;bra
 inherit ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'systemd', '', d)}
 inherit ${@bb.utils.contains('DISTRO_FEATURES', 'sysvinit', 'update-rc.d', '', d)}
 
+# tedge-p11-server is defined in a seprated recipe
+TEDGE_EXCLUDE += "tedge-p11-server"
+
 do_install:append () {
     if ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}; then
 		install -d ${D}${systemd_system_unitdir}
         
         for service in ${WORKDIR}/tedge-services/services/systemd/system/*; do
-            filename=$(basename "$service" ".service" | cut -d. -f1)
-            if ! echo "${@d.getVar('TEDGE_EXCLUDE')}" | grep -iq $filename; then
+            base=$(basename "$service")
+            unit_name=$(echo "$base" | cut -d. -f1)
```

Comment:
Since we're not using the `base` variable anywhere we could just put a one-liner.

```
unit_name=$(basename "$service" | cut -d. -f1)
```

Reply:
fixed in d43da11796b79206a2d1f1fb802e588d64faced7